crates/common/download/src/download.rs

```diff
@@ -231,7 +261,8 @@ impl Downloader {
     /// [`download_remaining`](Downloader::download_remaining) is used instead.
     async fn retry(&self, url: &DownloadInfo, file: &mut File) -> Result<(), DownloadError> {
         loop {
-            let mut response = request_range_from(url, 0).await?;
+            let mut response = self.request_range_from(url, 0).await?;
+            info!("Could not resume download, restarting");
```

Comment:
Surprised to see the `info!()` after and not before the request call.

Reply:
You're right, the log should happen before. Will fix.